neo4j/pom.xml

```diff
@@ -65,10 +63,9 @@
     </dependency>
 
     <dependency>
-      <groupId>org.neo4j.test</groupId>
-      <artifactId>neo4j-harness</artifactId>
-      <version>${neo4j.version}</version>
-      <scope>test</scope>
+      <groupId>org.testcontainers</groupId>
+      <artifactId>neo4j</artifactId>
+      <version>${testcontainers.version}</version>
```

Comment:
I don't think we add the version information here.

Reply:
We need it here because in the parent I only defined the `testcontainers` artifact:

https://github.com/apache/zeppelin/blob/4f2e95e5c694e2f2e786e9436af41c11a4726204/pom.xml#L921-L926

So every project can import the specific container, in the same way, I did for the `Neo4jContainer`.

Reply:
Oh, I've missed it. BTW, I think we'd better define neo4j version in the parent pom.xml as well and use it on this pom.xml without version information. I remembered that we already had these kinds of ways for others even if it's used in a single pom.xml only. WDYT?

Reply:
IMHO I prefer this approach because the Neo4j package is required only here, but if you think your is better as in the whole project is already adopted it's fine with me.
Let me know how I should proceed.